CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.2] - 2023-03-09
```

Comment:
Please remove this heading, it will be added automatically by the release workflow

Reply:
Thanks for the input, wasn't sure about this one :P Removed it